recipes/boolean-lite/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,23 @@
+#include "nonstd/boolean.hpp"
```

Comment:
@martinmoene 
The example at https://github.com/martinmoene/boolean-lite#example-usage is currently missing this include.

Reply:
@madebr
Thanks for your alertness.
I've corrected it. Also speculatively added the 'on conan' badge.